integration-tests/plugins/__tests__/regions/admin/regions.spec.ts

```diff
@@ -63,14 +65,21 @@ describe("Regions - Admin", () => {
         id: created.data.region.id,
         name: "Test Region",
         currency_code: "usd",
+        metadata: { foo: "bar" },
       })
     )
+    expect(created.data.region.countries.map((c) => c.iso_2).sort()).toEqual([
```

Comment:
comment: eventually you can also use arrayContaining to not have to sort. but I am fine with it